### PR TITLE
Make `StringUtil` classes static

### PR DESCRIPTION
**Pull request by xtqqczze**
Follow-up #14483.

### CODE REVIEW

**Review by iSazonov — APPROVED** [review]

File: src/Microsoft.PowerShell.ScheduledJob/ScheduledJobDefinition.cs [hunk 1]
```diff
@@ -2300,7 +2300,7 @@ internal string FQEID
     /// <summary>
     /// Simple string formatting helper.
     /// </summary>
-    internal class StringUtil
+    internal static class StringUtil
```
Comment (daxian-dbw) [inline src/Microsoft.PowerShell.ScheduledJob/ScheduledJobDefinition.cs:2303]: ScheduledJob module is not being built in the code base today, so please undo the change here.
Actually, we may need to think about remove the code that is not used today.

Comment (iSazonov) [reply src/Microsoft.PowerShell.ScheduledJob/ScheduledJobDefinition.cs:2303]: :-) LocalAccounts/StringUtil.cs too.

**Comment by iSazonov** [comment]
The code is not in compile.

File: src/Microsoft.PowerShell.ScheduledJob/ScheduledJobDefinition.cs (hunk 1 above)
Comment (xtqqczze) [reply src/Microsoft.PowerShell.ScheduledJob/ScheduledJobDefinition.cs:2303]: @daxian-dbw I guess work to remove uncompiled code is covered by #9960?
